### DISCUSSION TITLE
Error while starting Carla on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by rns0869** [carla-simulator/carla#5947] [Q&A]
I can't run Carla, when I run ./CarlaUE4.sh in the terminal I get an error.
My laptop has 8gb of RAM, RYZEN 5 4600H( 6 core and 12 thread Processor) and NVDIA GTX 1650 Ti graphics card.

Anyone has any solutions for this error?
The error is :

4.26.2-0+++UE4+Release-4.26 522 0
Disabling core dumps.
amdgpu: Not enough memory for command submission.
LowLevelFatalError [File:Unknown] [Line: 803] 
VulkanRHI::vkQueueSubmit(Queue, 1, &SubmitInfo, Fence->GetHandle()) failed, VkResult=-4
 at /home/jenkins/UnrealEngine_4.26/Engine/Source/Runtime/VulkanRHI/Private/VulkanQueue.cpp:71 
 with error VK_ERROR_DEVICE_LOST
Signal 11 caught.
Malloc Size=65538 LargeMemoryPoolOffset=65554 
CommonUnixCrashHandler: Signal=11
Malloc Size=131160 LargeMemoryPoolOffset=196744 
Malloc Size=131160 LargeMemoryPoolOffset=327928 
Engine crash handling finished; re-raising signal 11 for the default handler. Good bye.
Segmentation fault (core dumped)

**Comment by rreddy78** [comment]
Use the command line option -prefer-nvidia
